src/lib/viewers/controls/page/__tests__/PageControls-test.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PageControls from '../PageControls';
+import PageControlsForm from '../PageControlsForm';
+
+describe('PageControls', () => {
+    const getWrapper = (props = {}): ShallowWrapper =>
+        shallow(<PageControls onPageChange={noop} onPageSubmit={noop} pageCount={3} pageNumber={1} {...props} />);
+    const getPreviousPageButton = (wrapper: ShallowWrapper): ShallowWrapper =>
+        wrapper.find('[data-testid="bp-PageControls-previous"]');
+    const getNextPageButton = (wrapper: ShallowWrapper): ShallowWrapper =>
+        wrapper.find('[data-testid="bp-PageControls-next"]');
+
+    describe('event handlers', () => {
+        test('should handle previous page click', () => {
+            const pageNumber = 2;
+            const pageCount = 3;
+            const onPageChange = jest.fn();
+            const wrapper = getWrapper({ onPageChange, pageCount, pageNumber });
+
+            getPreviousPageButton(wrapper).simulate('click');
+
+            expect(onPageChange).toBeCalledWith(pageNumber - 1);
+        });
+
+        test('should handle next page click', () => {
+            const pageNumber = 2;
+            const pageCount = 3;
+            const onPageChange = jest.fn();
+            const wrapper = getWrapper({ onPageChange, pageCount, pageNumber });
+
+            getNextPageButton(wrapper).simulate('click');
+
+            expect(onPageChange).toBeCalledWith(pageNumber + 1);
+        });
+    });
+
+    describe('render', () => {
+        test.each`
+            pageNumber | pageCount | isPrevButtonDisabled | isNextButtonDisabled
```

Comment:
If `pageCount` doesn't change between tests, we can hardcode it in the test code.